### ISSUE TITLE
Just open src code.

### ISSUE THREAD

**Issue by catnikita255** [zhuowei/MCPELauncher#1428]
Just open source code. Latest version is a buggy shit. Lots of Minecrafters know Java and C/C++, release source code and we'll fix all the bugs. **JUST FUKEN OPEN IT, PLEASE!**

**Comment by zheka2304** [comment]
Yeah. This shit must be fixed, and if you cant do it, let us try. Too many people bought blocklauncher pro, even more use free version. And you just making a piece of shit in newest versions. 1.14.X versions just cannot handle big mods, that is something more than few new usual blocks and items.

**Comment by ISAAC300** [comment]
maybe not.

**Comment by DiegoGTRatty** [comment]
@catnikita255 @zheka2304 The point of Github is so you KINDA HELP OUT, instead of pressuring him to do this. Be reminded that he is doing this on his own time, and may decide to quit doing so if you keep this up.

**Comment by KenMizz** [comment]
do u know why zhuowei doesn't open the source code anymore?

**Comment by like2omg** [comment]
@KenMizz The reason is because of thieves stealing the code and making money on it before they can get shut down. (Once something goes online, it never will be able to be really removed, even if the place where it was hosted at removes it. This means, as an example, if YouTube videos can be dowloaded now, they can still exist after YT takes the video down).

**Comment by ISAAC300** [comment]
correct @like2omg
